Makefile

```diff
@@ -317,8 +317,8 @@ bios-flash: $(BIOS_FILE) bios-flash-$(PLATFORM)
 ATFTPD=$(shell which atftpd)
 ATFTPD?=/usr/sbin/atftpd
 
-IN_TFTPD=$(shell which in.tfptd)
-IN_TFTPD=?=/usr/sbin/in.tftpd
+IN_TFTPD=$(shell which in.tftpd)
+IN_TFTPD?=/usr/sbin/in.tftpd
```

Comment:
Note, I don't think this `?=` does what we expect since IN_TFTPD and ATFTPD vars are not evaluated until later.

Reply:
@stffrdhrn: Good catch.  ":=" is needed to force immediate evaluation ("=" is deferred it turns out), and "?=" actually only assigns "if not assigned" not (eg, like shell, perl) if "value is empty".  Reworked in https://github.com/timvideos/litex-buildenv/pull/17 which seems to work better (at least for `atftpd` case in testing here.  Thanks for the heads up.